common/src/main/java/com/microsoft/identity/common/internal/authorities/AADTestAuthority.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.identity.common.internal.authorities;
+
+import com.microsoft.identity.common.internal.logging.Logger;
+import com.microsoft.identity.common.internal.providers.microsoft.microsoftsts.MicrosoftStsOAuth2Configuration;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.ropc.ResourceOwnerPasswordCredentialsTestStrategy;
+
+import java.util.Map;
+
+public class AADTestAuthority extends AzureActiveDirectoryAuthority {
+
+    private static transient final String TAG = AADTestAuthority.class.getSimpleName();
+
+    public AADTestAuthority() {
+        // using organizations audience as common does not support ropc
+        super(new AnyOrganizationalAccount());
+    }
+
+    @Override
+    public OAuth2Strategy createOAuth2Strategy() {
+        final String methodName = ":createOAuth2Strategy";
+        Logger.verbose(
```

Comment:
nit: we might not need to log in test project. Adding comments is good enough imho.